stats/losses/core.py

```diff
@@ -10,6 +10,35 @@ def __init__(self, api_key):
         self.api_key = api_key
         print("created Config")  
 
+class StatsService():
+
+    def __init__(self):
+        API_KEY = os.getenv('RIOT_API_KEY_APP')
+        self.riotProxy = RiotProxy(API_KEY)
+
+    def calculate_dps_threats(self, names):
```

Comment:
res = [self.calculate_dps_threat(x) for x in names]

Reply:
Beautiful!